pkg/query/stat_processor.go

```diff
@@ -188,6 +194,70 @@ func (sp *defaultStatProcessor) process(workers uint) {
 	sp.wg.Done()
 }
 
+func generateQuantileMap(hist *hdrhistogram.Histogram) (int64, map[string]float64) {
+	ops := hist.TotalCount()
+	q0 := 0.0
+	q50 := 0.0
+	q95 := 0.0
+	q99 := 0.0
+	q999 := 0.0
+	q100 := 0.0
+	if ops > 0 {
+		q0 = float64(hist.ValueAtQuantile(0.0)) / 10e2
+		q50 = float64(hist.ValueAtQuantile(50.0)) / 10e2
+		q95 = float64(hist.ValueAtQuantile(95.0)) / 10e2
+		q99 = float64(hist.ValueAtQuantile(99.0)) / 10e2
+		q999 = float64(hist.ValueAtQuantile(99.90)) / 10e2
+		q100 = float64(hist.ValueAtQuantile(100.0)) / 10e2
+	}
+
+	mp := map[string]float64{"q0": q0, "q50": q50, "q95": q95, "q99": q99, "q999": q999, "q100": q100}
+	return ops, mp
+}
+
+func (b *BenchmarkRunner) GetOverallQuantiles(label string,histogram *hdrhistogram.Histogram) map[string]interface{} {
+	configs := map[string]interface{}{}
+	_, all := generateQuantileMap(histogram)
+	configs[label] = all
+	configs["EncodedHistogram"] = nil
+	encodedHist, err := histogram.Encode(hdrhistogram.V2CompressedEncodingCookieBase)
+	if err == nil {
+		configs["EncodedHistogram"] = encodedHist
+	}
+	return configs
+}
```

Comment:
Seems like an unused function, what is reason for adding it?

Reply:
@antekresic you're totally right. I've copied it from other tools we use and forgot to delete this unsed function.  addressed in latest commit